concordium-node/src/consensus_ffi/ffi.rs

```diff
@@ -1046,6 +1135,43 @@ impl ConsensusContainer {
         Ok((out_hash, out_data))
     }
 
+    /// Get the best guess as to what the next account sequence number should
+    /// be. If all account transactions are finalized then this information
+    /// is reliable. Otherwise this is the best guess, assuming all other
+    /// transactions will be committed to blocks and eventually finalized.
```

Comment:
```suggestion
    /// Get the best guess as to what the next account sequence number should
    /// be. If all account transactions are finalized, then this information
    /// is reliable. Otherwise, this is the best guess, assuming all other
    /// transactions will be committed to blocks and eventually finalized.
```